### PR TITLE
Features/group memberships controller

### PR DESCRIPTION
**Pull request by npauzenga**
Hey @Enriikke what do you think of this guy?

### CODE REVIEW

File: app/controllers/group_memberships_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class GroupMembershipsController < AuthenticationController
+  def create
+    result = CreateGroupMembership.call(user_id:  current_user.id,
+                                        group_id: params[:group_id])
+    if result.success?
+      head :no_content
```
Comment (npauzenga) [inline app/controllers/group_memberships_controller.rb:6]: @Enriikke `:no_content` seems the most logical? I think the front-end will just respond to a successful create with a redirect and a success message...

File: app/controllers/group_memberships_controller.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class GroupMembershipsController < AuthenticationController
+  def create
+    result = CreateGroupMembership.call(user_id:  current_user.id,
+                                        group_id: params[:group_id])
```
Comment (npauzenga) [inline app/controllers/group_memberships_controller.rb:4]: @Enriikke so here I'm not entirely sure how I'm going to get the `group_id`. Perhaps we could talk a bit about this...

File: app/controllers/group_memberships_controller.rb (hunk 1 above)
Comment (enriikke) [inline app/controllers/group_memberships_controller.rb:6]: That makes sense to me too. Usually you need to return something if you generated some new data that the client doesn't have. In this case the client already has everything needed.

File: app/controllers/group_memberships_controller.rb (hunk 2 above)
Comment (enriikke) [inline app/controllers/group_memberships_controller.rb:4]: I can think the endpoint could be something like:

```
POST /groups/:group_id/memberships
```

That will include the group id within the endpoint itself.

**Comment by enriikke** [comment]
You are doing great @npauzenga!!! :shipit: :8ball:

**Pull request merged**